.ci/publish-releasenotes-outside.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [[ -z $1 ]]; then
+  echo "latest version is not set"
+  exit 1
+fi
+
+TARGET_VERSION=$1
+echo TARGET_VERSION="$TARGET_VERSION"
+
+CURRENT_VERSION=$(xmlstarlet sel -N pom=http://maven.apache.org/POM/4.0.0 \
+                             -t -m pom:project -v pom:version pom.xml | sed "s/-SNAPSHOT//")
+echo CURRENT_VERSION="$CURRENT_VERSION"
+
+if [ "$TARGET_VERSION" != "$CURRENT_VERSION" ]; then
+  echo "[ERROR] Target Version and Current Version doesn't match."
+  exit 1;
+fi
+
+# Publish Tweet
+./.ci/tweet-releasenotes.sh "$TARGET_VERSION"
```

Comment:
lets move this execution to separate Github action (I know that it will be some copy-paste), as it will require extra security values to be provided.

Reply:
@romani I believe that we should put this at script only as the maintainer can run it locally too!

Reply:
Changes Done

Reply:
.ci/publish-releasenotes-outeside.sh --rename--> ci/publish-releasenotes-githubpage.sh (and remove weet-releasenotes.sh)

in separate PR make new gitub action (it will copy of this action), that use  "copy" of .ci/publish-releasenotes-outeside.sh that does only gitub  page update. And it will special security variables.